### PR TITLE
Kai Yang bio

### PR DESCRIPTION
**Pull request by HKYang**
add Kai Yang's bio

### CODE REVIEW

**Comment by gvwilson** [comment]
Hi @HKYang ,
In order to be stitched into the blog correctly, posts need to have a header with values like the date/time of publication, the author's name, etc.  Please see `_submissions/round-12/0/2015-02-01-about-greg-wilson.md` for an example - if you add a header like that to your file and push to your repository again, this pull request will automatically update.
Thanks,
Greg

**Comment by HKYang** [comment]
Done. Please verify or notify any missing step.
Thanks.
-kai

File: _submissions/round-12/0/2015-02-06-about-kai-yang.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+date: 2014-02-09
```
Comment (gvwilson) [inline _submissions/round-12/0/2015-02-06-about-kai-yang.md]: Please change the year to 2015-02-09 (this year).

Comment (HKYang) [inline _submissions/round-12/0/2015-02-06-about-kai-yang.md]: Done. Please verify. Thanks.
-kai

**Comment by HKYang** [comment]
update date

**Pull request closed without merging**